app/assets/stylesheets/application.scss

```diff
@@ -32,3 +32,56 @@
   position: static;
   width: 100%;
 }
+
+.right_text{
+  float: right;
+}
+
+.container {
+  h1 {
+    color: white;
+    font: bold 24px/45px Helvetica, Sans-Serif;
+    letter-spacing: 6px;
+    background: rgba(0, 0, 0, 0.6);
+    padding: 10px;
+    font-variant: small-caps;
+  }
+}
+
+h4{
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered background, color, font, font-variant, letter-spacing, padding